nata/simpleBlogHomePage/index.html

```diff
@@ -1,10 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <title>Simple Blog Home Page</title>
     <link rel="stylesheet" href="styles.css" />
-  </head>
-  <body>
-  </body>
-</html>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <a href="./"><img src="./Img/dasdas.jpg" alt="Dasdas Logo" width="80px" height="80px" /></a>
+        <nav>
+            <div class="icons">
+                <a href="https://www.twitter.com/">
+                    <img src="./Img/twitter.png" alt="Twitter" width="20px" height="20px" /></a>
+
+                <a href="https://www.dribbble.com/">
+                    <img src="./Img/dribbble.png" alt="Dribbble" width="20px" height="20px" /></a>
+
+                <a href="https://www.figma.com/">
+                    <img src="./Img/figma.png" alt="Figma" width="20px" height="20px" /></a>
+
+                <a href="https://www.github.com/">
+                    <img src="./Img/github.png" alt="Github" width="20px" height="20px" /></a>
```

Comment:
1. All images are blurry, just remove width and height attributes

```suggestion
                <a href="https://www.twitter.com/">
                    <img src="./Img/twitter.png" alt="Twitter" /></a>

                <a href="https://www.dribbble.com/">
                    <img src="./Img/dribbble.png" alt="Dribbble" /></a>

                <a href="https://www.figma.com/">
                    <img src="./Img/figma.png" alt="Figma" /></a>

                <a href="https://www.github.com/">
                    <img src="./Img/github.png" alt="Github" /></a>
```

2. Please rename `Img` folder just as `img`